### PR TITLE
Tv/feat/#41

### PR DESCRIPTION
**Pull request by thaysvieira**

### CODE REVIEW

File: carRental/src/main/java/com/amigoscode/chohort2/carRental/carProviderAddress/CarProviderAddressDTO.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class CarProviderAddressDTO implements Serializable {
+    private Long id;
+    private Long carProviderId;
```
Comment (Esmaeeilenani) [inline carRental/src/main/java/com/amigoscode/chohort2/carRental/carProviderAddress/CarProviderAddressDTO.java:15]: remove line 16

File: carRental/src/main/java/com/amigoscode/chohort2/carRental/carProviderAddress/CarProviderAddress.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import org.hibernate.annotations.Where;
+
+@Entity
+@Table(name = "car_provider_address")
+@Setter
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+@Accessors(chain = true)
+@Where(clause = "is_deleted = false")
```
Comment (Esmaeeilenani) [inline carRental/src/main/java/com/amigoscode/chohort2/carRental/carProviderAddress/CarProviderAddress.java:21]: should extends AbstractAddress

**Review by DmitriKonnovNN — COMMENTED** [review]
awaiting changes required by Esmaeeil

**Review by Esmaeeilenani — APPROVED** [review]

File: carRental/src/main/java/com/amigoscode/chohort2/carRental/carProviderAddress/CarProviderAddressDTO.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    private Long cityId;
+    private CityDTO cityDTO;
```
Comment (DmitriKonnovNN) [inline carRental/src/main/java/com/amigoscode/chohort2/carRental/carProviderAddress/CarProviderAddressDTO.java:17]: Needs change to: **private CityDTO city;**

**Review by DmitriKonnovNN — CHANGES_REQUESTED** [review]

**Review by DmitriKonnovNN — APPROVED** [review]
Look absolutely fantastic

**Pull request merged**